src/layouts/base-layout.astro

```diff
@@ -17,7 +17,7 @@ const { title } = Astro.props;
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link href="favicon.svg" rel="icon" type="image/svg+xml" />
+    <link href="/favicon.svg" rel="icon" type="image/svg+xml" />
```

Comment:
note: this change was made in order to fix relative path lookup (instead of lookup to the public directory). not sure if this is only a development issue, but worth noting